src/libutil-tests/hash.cc

```diff
@@ -10,6 +10,19 @@ namespace nix {
      * hashString
      * --------------------------------------------------------------------------*/
 
```

Comment:
There doesn't seem to be tests added for the SRI hash formats.

Reply:
I'm not sure if it's necessary to have these since the SRI parsing code is independent of the hashing algorithm code. I guess this is why the other algorithms didn't include these either. But I did add separate tests for these in a new commit.